app/config/parameters.yml.dist

```diff
@@ -22,6 +22,9 @@ parameters:
     # with MySQL, use "utf8mb4" if you got problem with content with emojis
     database_charset: utf8
 
+    # Instance URL
```

Comment:
We already store the wallabag URL in the application. If we define a new parameter here, I think we should remove the other one.